src/main/java/zeenea/connector/ConnectionConfiguration.java

```diff
@@ -85,6 +87,29 @@ public interface ConnectionConfiguration {
    */
   FilterConfiguration getFilters();
 
+  /**
+   * Gets a Map associated with the specified key.
+   *
+   * <p>Example: environment.variables { "DEFAULT_WAREHOUSE":"zeenea_wh",
```

Comment:
Comme ça plutôt ? J'ai pas testé je sais pas comment ça s'utilise.
```suggestion
   * <p>Example: environment.variables = { "DEFAULT_WAREHOUSE":"zeenea_wh",
```

Reply:
je vais re tester mais je crois que c'est K.O

Reply:
ça fonctionne mais ce n'est pas écrit comme ça dans la doc :
https://docs.spongepowered.org/stable/fr/server/getting-started/configuration/hocon.html
ni chatgpt :

_Comment je peux créer l'équivalent en HOCON d'un Map<String, String> ?_


> ChatGPT a dit :
> Bonne question ! En HOCON, l'équivalent d’un Map<String, String> en Java, c’est simplement un objet avec des paires clé/valeur où les clés et les valeurs sont des chaînes de caractères.
> 
> Exemple HOCON équivalent à Map<String, String> :
> hocon
> Copier
> Modifier
> maMap {
>   cle1 = "valeur1"
>   cle2 = "valeur2"
>   cle3 = "valeur3"
> }

Reply:
Ah oui c'est du hocon, bien sûr. Donc oui ça marche comme ça, merci !